assets/js/blocks/classic-template/archive-product.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import {
+	getBlockType,
+	createBlock,
+	createBlocksFromInnerBlocksTemplate,
+	type BlockInstance,
+} from '@wordpress/blocks';
+import { isWpVersion } from '@woocommerce/settings';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { PLACEHOLDERS } from './constants';
+import {
+	INNER_BLOCKS_TEMPLATE as productsInnerBlocksTemplate,
+	QUERY_DEFAULT_ATTRIBUTES as productsQueryDefaultAttributes,
+} from '../product-query/constants';
+import { VARIATION_NAME as productsVariationName } from '../product-query/variations/product-query';
+
+const createRowBlock = ( innerBlocks: Array< BlockInstance > ) => {
+	const rowVariationName = `group-row`;
+	const groupBlockVariations = getBlockType( 'core/group' )?.variations || [];
+	const rowVariation = groupBlockVariations.find(
+		( { name }: { name: string } ) => name === rowVariationName
+	);
+
+	if ( ! rowVariation ) {
+		return null;
+	}
+
+	const { attributes } = rowVariation;
+	const extendedAttributes = {
+		...attributes,
+		layout: {
+			...attributes.layout,
+			justifyContent: 'space-between',
+		},
+		align: 'wide',
```

Comment:
We're hardcoding the value of align here to `wide`. What if the merchant changes the alignment of the Classic Template block to `None` or `Full width`? Can we preserve that attribute?

Reply:
Great point! `align` attribute is now inherited from the placeholder block ([commit](https://github.com/woocommerce/woocommerce-blocks/pull/8248/commits/36f740b18c8ecebd6e14d7694ea5ad38e35b2e43) - but note that this evolved a bit in next commits).